app/src/github_runner_image_builder/templates/cloud-init.sh.j2

```diff
@@ -128,7 +150,9 @@ install_apt_packages "$apt_packages" "$hwe_version"
 disable_unattended_upgrades
 enable_network_fair_queuing_congestion
 configure_usr_local_bin
-install_yarn
+if [ $RELEASE != "focal" ]; then
```

Comment:
Does focal come with yarn? Would you be able to add a comment here as well explaining the rationale behind this as well please? Thank you!

Reply:
ah nvm. saw the docs below - but perhaps also worth a comment!